src/config.py

```diff
@@ -651,4 +651,35 @@ def from_network(network: Network) -> AccountingConfig:
         )
 
 
+@dataclass(frozen=True)
+class OverdraftConfig:
+    """Full configuration for the overdrafts management contract"""
+
+    contract_address: str
```

Comment:
I would use a more descriptive name, e.g., overdrafts_contract_address.

Also, as all addresses in `config.py` use the `Address` data type, I would also use that one here.